functions/openhab.bash

```diff
@@ -267,10 +267,10 @@ dashboard_add_tile() {
   application="$1"
   openhabConfig="/etc/openhab"
   dashboardConfig="${openhabConfig}/services/runtime.cfg"
-  ipAddress="$(hostname -I)"
+  ipAddress="$(ip route get 8.8.8.8 | head -1 | grep -Po '(?<=src )(\d{1,3}.){4}' | xargs)"
```

Comment:
Nice one, you're never done learning.
But given the manpage says (below), could you make it a little less experimental ?

 -P, --perl-regexp
              Interpret PATTERNS as Perl-compatible regular expressions (PCREs).  This option is experimental when combined with the -z (--null-data) option, and grep -P may
              warn of unimplemented features.

Reply:
I'll see what I can do.

Reply:
@mstormi What about now?